tcl/pd-gui.tcl

```diff
@@ -114,9 +114,8 @@ set PD_BUGFIX_VERSION 0
 set PD_TEST_VERSION ""
 set done_init 0
 
-set TCL_MAJOR_VERSION 0
-set TCL_MINOR_VERSION 0
-set TCL_BUGFIX_VERSION 0
+# this complements tcl_version by providing the patchlevel version aka #.#.patch
+set TCL_PATCHLEVEL 0
```

Comment:
thanks for removing global variables.
please don't introduce new globals that are practically unused.

Reply:
I *have* to know what the patch level is on macOS as sending negative positions to the buggy TK 8.5.9 causes a crash. It's either this or a local variable within pdtk_canvas.tcl or running scan on [info patchlevel] every time a window is opened which I figured was a waste of resources when the value can be fetched ahead of time.